cclibs/styles/styles.css

```diff
@@ -6,3 +6,9 @@
  * 
  *
  */
+
+ .reading-width {
+  max-width: 600px;
+  margin: auto;
+  padding: 0 20px;
+}
```

Comment:
I don't think this should be here.

Reply:
@aishwaryamathuria If it's authored by section-metadata then the css should at the very least be in your section-metadata.css

Reply:
@aishwaryamathuria where do you need this css to take effect? If you need this for short form articles, this is probably a change you'd want to make directly there. This CSS doesn't belong in the styles.css

Reply:
@vhargrave The short-form articles have "reading width" specified as a style in section block. So currently it requires any section with reading-width class set to be formatted with this style.